services/public-api/internal/migrations/2_graph.sql

```diff
@@ -0,0 +1,28 @@
+-- +migrate Up
+
+CREATE TABLE nodes (
+    nodeId varchar(255) UNIQUE,
+    doc varchar(255),
+    edgesOut varchar(255)[],
+    edgesIn varchar(255)[],
```

Comment:
I need to look into postgres querying and get a better sense of how arrays affect performance.